src/sources/prometheus/parser.rs

```diff
@@ -830,20 +880,20 @@ mod test {
         let exp = r##"
             # HELP nginx_server_bytes request/response bytes
             # TYPE nginx_server_bytes counter
-            nginx_server_bytes{direction="in",host="*"} 263719
```

Comment:
Just noticing that now this example doesn't match the example in the issue linked in the comment for this test.

Reply:
Would you recommend I change the test data back to what is in the incident (which will prevent comparison for whole event equality), or just drop the link to the incident?

Reply:
Ah, yeah I see. I guess I'd ideally leave the example as it was from the issue and then mock `now` and assert that, similar to how you did in #6823 .

Reply:
Fixed.